### PR TITLE
Add a command to fetch the UID of the device.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/message_handler.c [hunk 1]
```diff
@@ -73,6 +75,19 @@ static void SetMode(uint8_t token,
   SendMessage(token, COMMAND_SET_MODE, RC_OK, NULL, 0);
 }
 
+static void GetUID(uint8_t token, unsigned int length) {
+  if (length) {
+    SendMessage(token, COMMAND_SET_MODE, RC_BAD_PARAM, NULL, 0);
```
Comment (peternewman) [inline firmware/src/message_handler.c]: Should this be COMMAND_GET_MODE?

Comment (peternewman) [inline firmware/src/message_handler.c]: Or even COMMAND_GET_UID

**Comment by peternewman** [comment]
Just one comment

File: firmware/src/message_handler.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/message_handler.c]: Fixed.

**Pull request merged**